src/Frontend/Modules/FormBuilder/Widgets/Form.php

```diff
@@ -523,21 +547,33 @@ private function validateForm()
                         )
                         ->setTo($this->item['email'])
                         ->setFrom(array($from['email'] => $from['name']))
-                    ;
+                        ->setReplyTo(array($replyToAddress => $replyToName));
 
-                    // check if we have a replyTo email set
-                    foreach ($this->item['fields'] as $field) {
-                        if (array_key_exists('reply_to', $field['settings']) &&
-                            $field['settings']['reply_to'] === true
-                        ) {
-                            $email = $this->frm->getField('field' . $field['id'])->getValue();
-                            $message->setReplyTo(array($email => $email));
-                        }
-                    }
-                    if ($message->getReplyTo() === null) {
-                        $replyTo = FrontendModel::getModuleSetting('Core', 'mailer_reply_to');
-                        $message->setReplyTo(array($replyTo['email'] => $replyTo['name']));
-                    }
+                    $this->get('mailer')->send($message);
+                }
+
+                // Confirmation e-mail to person who filled in the form
+                // Double check for empty subject or content and make sure the e-mail is allowed to be send.
+                // Also, make sure we have a valid reply to e-mail adres (which is checked above)
+                if (
+                    trim($this->item['confirmation_mail_subject']) != '' &&
+                    strip_tags(trim($this->item['confirmation_mail_content'])) != '' &&
+                    $this->item['send_confirmation_mail'] == 'Y' &&
+                    $replyToGiven
+                ) {
+                    $message = \Common\Mailer\Message::newInstance($this->item['confirmation_mail_subject'])
```

Comment:
The fully qualified classname `\Common\Mailer\Message` is used a lot in this file, would be better if there was a `use` statement